modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementFailureTest.java

```diff
@@ -454,4 +454,33 @@ public void testGetPaginatedOrganizationsDiscoveryWithInvalidOffsetAndLimitZero(
         validateErrorResponse(response, HttpStatus.SC_INTERNAL_SERVER_ERROR, ERROR_CODE_SERVER_ERROR);
     }
 
+    @Test(dependsOnMethods = "testGetPaginatedOrganizationsDiscoveryWithInvalidLimitAndOffset")
+    public void testGetPaginatedMetaAttributesWithInvalidLimit() {
+
+        String invalidLimitUrl =
+                ORGANIZATION_MANAGEMENT_API_BASE_PATH + ORGANIZATION_META_ATTRIBUTES_API_PATH + QUESTION_MARK +
+                        LIMIT_QUERY_PARAM + EQUAL + "-1";
```

Comment:
```
                ORGANIZATION_MANAGEMENT_API_BASE_PATH + ORGANIZATION_META_ATTRIBUTES_API_PATH + QUESTION_MARK +
                        LIMIT_QUERY_PARAM + EQUAL;
```
There are multiple places where we use this combination. Shall we create a single constant for this?